CMakeLists.txt

```diff
@@ -25,13 +25,27 @@ project(opencensus-cpp VERSION 0.3.0 LANGUAGES CXX)
 
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(OPENCENSUS_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
+
+option(BUILD_SHARED_LIBS "Build shared libraries" OFF)
+
+IF(MSVC)
+    add_definitions(-DNOMINMAX)
+ENDIF()
+
+if(CMAKE_BUILD_TYPE STREQUAL "Release")
```

Comment:
This is unusual... is that to avoid installing the headers twice?

Reply:
> 
> 
> @meastp: Would you like to split out `Replace OPENCENSUS_INCLUDE_DIR with CMAKE_SOURCE_DIR` into its own PR? (and please convert to Unix line endings) If that doesn't break an out-of-tree example, I'd be happy to merge that.

Sure :)

Reply:
I think so. To be honest I took that example from somewhere else. What is the proper way to do it? :)